### DISCUSSION TITLE
Do not release Black Temple mythic and normal at the same time.

### DISCUSSION THREAD

**Discussion by TotallyNotMyAccount** [chromiecraft/chromiecraft#7370] [Suggestions]
For anyone in the race for server first, releasing both difficulties at once is going to create uncomfortable choices about what to prioritize. Lets please just sidestep all the stupid meta considerations, so people can give their best and enjoy themselves on release night.

I'd personally prefer if normal only released the ID after mythic was cleared the first time, but the other way around would still be far better than both releasing at once.

**Comment by Belodina** [comment]
I agree with the other way around. I think Black Temple Mythic should if possible, unlock from the second week of Raid release, or after the server first Normal-mode Illidan kill.

This would be a healthy change to prevent inter-guild drama so all guilds are on an equal understanding and level playing field in regards to competition over both server first kills. 

This is also similar to how Heroic TGC and ICC modes were released after the normal versions

**Reply by Never6** [reply]
this makes more sense

**Comment by BeefSteaker** [comment]
Surely people can just try mythic on release night for world first, and if they change their mind lower it to normal?

groups should be able to reasonably predict if they can do mythic by tesing on the PTR. i would imagine any guild with a chance of world first will do plenty of PTR prep

**Reply by shaszun** [reply]
> Surely people can just try mythic on release night for world first, and if they change their mind lower it to normal?
> 
> groups should be able to reasonably predict if they can do mythic by tesing on the PTR. i would imagine any guild with a chance of world first will do plenty of PTR prep

Players did a lot of reasonable predictions for the past few brackets and staff didn't listen once. We know the outcomes of each one of them. By world first, you mean 1 guild at this moment on the server. Every other guild lost the willingness to even test stuff on the PTR. If you ask what do I mean? I mean the 'pug' raid that was formed by 5+ guilds to check BT. Also from past experience tuning always changed on release compared to PTR...

**Reply by TotallyNotMyAccount** [reply]
If mythic is tuned in a way that makes clearing it on day one impossible and clearing ID one very hard(hopefully to both), then you are giving up the day one achievement if you go for mythic. There's also the minor prestige of having cleared normal first. Is it a worthwhile trade, if you can get realm first mythic instead? Probably, but the achievement smarts for sure.
I really don't get why we should have to make choices like that at all.

**Comment by krutoj87** [comment]
Isn't there only one guild right now "racing" to server first? In this case the race can hardly be the problem. Why don't you explain what the real problem is?

**Reply by Belodina** [reply]
Please refrain from passive-aggressive comments. The point is to be constructive and gather genuine sentiment to the idea of staggering the releases to allow guilds the chance to evenly compete for both server first kills, even if only the Mythic kill is recognized/rewarded. 

I think the players of the other large raiding guilds would disagree, they are easily just as capable of winning server first, and there will definitely be multiple guilds clearing the first night as fast as they can. This holds true for whether it is Mythic or Normal, even more so now there is an additional reward for the Mythic server first clear, although the normal clear still holds merit (just as it would have done if there were no Mythic mode).

**Reply by krutoj87** [reply]
I don't think my comment was passive aggressive. Actually I think I was very direct. To me it just seems like the only guild caring about server firsts is Solaris and all the other guilds just go on with their normal raiding schedule and don't even try to clear the content first. 

If I'm wrong somebody from another guild can just tell me, that they also plan participate in the race for server first.

**Reply by Never6** [reply]
> I think the players of the other large raiding guilds would disagree, they are easily just as capable of winning server first, and there will definitely be multiple guilds clearing the first night as fast as they can. This holds true for whether it is Mythic or Normal, even more so now there is an additional reward for the Mythic server first clear, although the normal clear still holds merit (just as it would have done if there were no Mythic mode).

not really, as stated their is only one guild that even somewhat cares about server first.

**Reply by Belodina** [reply]
Hey @Never6 I can't speak for other guilds of course, but it doesn't make sense to me that guilds just wouldn't bother with it, especially now with recognition for the server-first kill. I know of course this topic in particular is something that riles up the community and because of it Solaris specifically is not well-liked. 

However, it would be sad to assume that no other guild cares or would even bother with it. Among other things, the state of LegacyPlayers should be addressed so there can actually be a healthy parsing/speedrunning community on the server, but due to the developmental nature this is somewhat difficult to achieve.

Regarding the actual suggestion of staggering releases, I personally will not even be able to raid on October 9th and agree with Choof that the Mythic server first will eclipse whoever kills Illidan on normal first. But as of yet, the mythic tuning is undetermined, and I can empathize with wanting a staggered release, so that an "equal" progression race for both normal and mythic server first can take place.

As far as I know, releasing both versions at the same time is more Blizzlike, and I'll be happy with however the raid is released as long as the state is good and players are happy.


To @krutoj87 No harm meant, I personally found your comment passive-aggressive, and I too would like to know if other Guild Masters are planning on competing for server first normal or mythic. Again, this is just a suggestion and if not well received is not likely to be implemented, hence the importance of being constructive :P The post is clear that this suggestion is to prevent guilds having to prioritize on whether they have to compete for server first Normal or Mythic. Since by starting Mythic and changing to Normal halfway through you have slowed yourself down such that you are not likely to get the server first Normal Kill, while if you choose to focus on the Normal kill you have to lock yourself out of attempting server first Mythic. Staggering the releases means every guild that wants to compete for both for example (which we do not know is only Solaris yet, as tuning is undetermined yet) will get an even chance to.

**Reply by heyitsbench** [reply]
Most guilds I'm sure don't have the luxury of being able to move their raid times around to be able to compete in such a race on this server.

**Comment by UndoUreche** [comment]
I'm very much undecided about which I'd like to see released first but either way, they shouldn't be released at the same time. Support this I do.

**Comment by walkingminterly** [comment]
Valens should come back and create a new Guild to compete for Server firsts, noone else cares except Solaris
